tests/nsinker_bfbt.prm

```diff
@@ -0,0 +1,14 @@
+set Dimension = 3
+
+include $ASPECT_SOURCE_DIR/benchmarks/nsinker/nsinker.prm
+
+subsection Solver parameters
+   subsection Stokes solver parameters
+   set Use weighted BFBT for Schur complement          = true
+
+  end
+end
+subsection Mesh refinement
+  set Initial adaptive refinement              = 0
+  set Initial global refinement                = 1
+end
```

Comment:
Like Timo commented above: keep this test, but also make a new nsinker_bfbt benchmark .prm that looks more or less like this test (including the original benchmark and switching to bfbt).